new_samples/workflows/types.go

```diff
@@ -0,0 +1,6 @@
+package workflows
```

Comment:
no "types" files please.  there's really no reason to do it, it just hides the type's contents behind a file-navigation that doesn't need to be done, stick it in the workflow.go or something

Reply:
sure. Also discussed with Ender offline we will use more vivid examples. So I'll go ahead delete this and put it inside the hello_world_workflow.go.

Reply:
"vivid"?